origin.spec

```diff
@@ -413,6 +418,10 @@ fi
 
 %post node
 %systemd_post %{name}-node.service
+# If accounting is not currently enabled systemd reexec
+if [[ `systemctl show docker %{name}-node | grep -q -e CPUAccounting=no -e MemoryAccounting=no` == 0 ]]; then
```

Comment:
In general, `command` does not return return code. `systemctl show docker %{name}-node | grep -q -e CPUAccounting=no -e MemoryAccounting=no` always returns an empty string. Which is then (most likely) converted to 0. So the condition will be always true. The only I am aware to test for RC is to check ``$?``.